src/validation.cpp

```diff
@@ -545,6 +546,7 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
 {
     const CTransaction& tx = *ptx;
     const uint256 hash = tx.GetHash();
+    CFeeRate poolMinFeeRate;
```

Comment:
In commit "Add mempool statistics collector"

Would be good to move declaration closer to where this is being set (at least as close as possible).